.circleci/config.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           name: Test Library
           command: yarn test:ci
 
-  install-test-example:
+  install-test-example-apps:
```

Comment:
I think `example` should be removed here

Reply:
done